package.json

```diff
@@ -2,47 +2,53 @@
   "name": "iter-tools",
   "version": "4.0.0",
   "description": "The iterator toolbox",
-  "main": "lib/index.js",
-  "module": "lib/index.mjs",
-  "typings": "lib/index.d.ts",
+  "main": "src/index.js",
+  "module": "src/index.mjs",
+  "typings": "src/index.d.ts",
   "scripts": {
     "test": "npm run build && mocha tests/**/*.js",
-    "benchmarks": "node benchmarks",
+    "benchmarks": "npm run build && node benchmarks",
     "watch": "npm run test -- -w",
     "lint": "eslint --fix --ext .mjs ./src ./tests ./benchmarks",
-    "clean": "rm -rf lib/*.* && rm -rf es5/*.* && rm -rf package/*",
-    "build": "npm run clean && cross-env BABEL_ENV=es6 babel -x \".mjs\" src -d lib && renamer --regex --find '\\.js$' --replace '.mjs' 'lib/**' && cross-env BABEL_ENV=es5 babel -x \".mjs\" src -d es5 && cross-env BABEL_ENV=cjs babel -x \".mjs\" src -d lib && cp src/*.d.ts lib",
-    "dist": "npm run build && cp -R package.json lib/* es5 LICENSE README.md package/ && cd package && json -I -f package.json -e \"this.main='index.js';this.typings='index.d.ts';this.module='index.mjs'\" && npm publish",
+    "clean": "rimraf dist",
+    "build": "npm run clean && npm run build:cjs && npm run build:es5 && npm run build:es5-cjs && npm run build:es2015 && npm run build:es2015-cjs && cp src/*.d.ts dist",
+    "build:cjs": "cross-env BABEL_ENV=cjs babel -x \".mjs\" src -d dist && copyfiles -f src/*.mjs dist/",
+    "build:es5": "cross-env BABEL_ENV=es5 babel -x \".mjs\" src -d dist/es5 && renamer --regex --find '\\.js$' --replace '.mjs' 'dist/es5/**'",
+    "build:es5-cjs": "cross-env BABEL_ENV=es5-cjs babel -x \".mjs\" src -d dist/es5",
+    "build:es2015": "cross-env BABEL_ENV=es2015 babel -x \".mjs\" src -d dist/es2015 && renamer --regex --find '\\.js$' --replace '.mjs' 'dist/es2015/**'",
```

Comment:
You can get avoid having to user renamer since you've brought in babel 7. You can just use `babel --keep-file-extension` instead.